changes/gem5-resources.tex

```diff
@@ -0,0 +1,13 @@
+\subsection[gem5 resources]{gem5 resources\footnote{by Mahyar Samani}}
+
+gem5 resources are the set of inputs used to conduct computer systems architecture
```

Comment:
I might rephrase the introduction to something like: "gem5 resources consist of a set of components used to perform certain simulation experiments (e.g., Linux boot tests) with gem5."  (just a suggestion)